homework/index.js

```diff
@@ -30,14 +30,129 @@
     return elem;
   }
 
+  function createOptions(repositories, select) {
+    repositories.forEach((repository, index) => {
+      createAndAppend('option', select, {
+        text: repository.name,
+        value: index,
+      });
+    });
+  }
+
+  function addRow(repository, table, property) {
+    const tbody = createAndAppend('tbody', table);
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('th', tr, {
+      text: `${property.toUpperCase()}: `,
+    });
+    if (property !== 'name') {
+      createAndAppend('td', tr, {
+        text: `${repository[property]}`,
+        id: property,
+      });
+    } else {
+      const firstRow = createAndAppend('td', tr);
+      createAndAppend('a', firstRow, {
+        text: `${repository[property]}`,
+        href: repository.html_url,
+      });
+    }
+  }
+
+  function renderRepositoryBasicInfo(repository, table) {
+    table.innerHTML = '';
+    addRow(repository, table, 'name');
+    addRow(repository, table, 'description');
+    addRow(repository, table, 'forks');
+    addRow(repository, table, 'updated_at');
+    const upDate = document.getElementById('updated_at');
+    upDate.innerHTML = '';
+    const dateArray = repository.updated_at.split('T');
+    upDate.innerHTML = `<em>${dateArray[0]}</em>, ${dateArray[1].substring(
```

Comment:
This is a style break with the rest of the code. We are creating all elements with `createAndAppend`. For consistency, you should do the same here. You could an `<em>` element like this:

```js
const datePart = createAndAppend('em', update, {text: dateArray[0]});
```

But then again, I would not bother trying to emphasize the date part at all.